beam/src/main/java/org/apache/zeppelin/beam/StaticRepl.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.beam;
+
+import javax.tools.Diagnostic;
+import javax.tools.DiagnosticCollector;
+import javax.tools.JavaCompiler;
+import javax.tools.JavaCompiler.CompilationTask;
+import javax.tools.JavaFileObject;
+import javax.tools.SimpleJavaFileObject;
+import javax.tools.ToolProvider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.thoughtworks.qdox.JavaProjectBuilder;
+import com.thoughtworks.qdox.model.JavaClass;
+import com.thoughtworks.qdox.model.JavaSource;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+import java.io.StringReader;
+import java.lang.reflect.InvocationTargetException;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * 
+ * StaticRepl for compling the java code in memory
+ * 
+ */
+public class StaticRepl {
+  static Logger logger = LoggerFactory.getLogger(StaticRepl.class);
+
+  public static String execute(String generatedClassName, String code) throws Exception {
+
+    JavaCompiler compiler = ToolProvider.getSystemJavaCompiler();
+    DiagnosticCollector<JavaFileObject> diagnostics = new DiagnosticCollector<JavaFileObject>();
+
+    // Java parasing
+    JavaProjectBuilder builder = new JavaProjectBuilder();
+    JavaSource src = builder.addSource(new StringReader(code));
+
+    // get all classes in code (paragraph)
+    List<JavaClass> classes = src.getClasses();
+    String mainClassName = null;
+
+    // Searching for class containing Main method
+    for (int i = 0; i < classes.size(); i++) {
+      boolean hasMain = false;
+
+      for (int j = 0; j < classes.get(i).getMethods().size(); j++) {
+        if (classes.get(i).getMethods().get(j).getName().equals("main")) {
+          mainClassName = classes.get(i).getName();
```

Comment:
I mean, this is not currently checking if the returned method is a static method?
something like `Modifier.isStatic`?